federator/package.json

```diff
@@ -12,11 +12,16 @@
   ],
   "author": "IOVLabs",
   "scripts": {
-    "start": "node ./src/main.js",
+    "start": "node ./built/federator/src/main.js",
+    "start-js": "node ./src/main.js",
+    "build-start": "npm run build && npm run start",
+    "build": "npm run clean && tsc",
```

Comment:
is typescript already installed when running `npm install` for the federator?

Reply:
nop, my bad 
I forgot to run `npm i typescript --save-dev` since I have it enable globally :/

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/253/commits/27f0562d0bd4e77f6d4ed22e796051d6c62ab0be